bazel_tools/haskell-optp-response.patch

```diff
@@ -0,0 +1,32 @@
+Some targets use a lot of space for -optP flags, so put them in a response file
+diff --git a/haskell/private/actions/compile.bzl b/haskell/private/actions/compile.bzl
+index 20c8720..7fca51c 100644
+--- a/haskell/private/actions/compile.bzl
++++ b/haskell/private/actions/compile.bzl
+@@ -213,7 +213,17 @@ def _compilation_defaults(hs, cc, java, dep_info, plugin_dep_info, srcs, import_
+             set.mutable_insert(import_dirs, idir)
+
+     ghc_args += ["-i{0}".format(d) for d in set.to_list(import_dirs)]
+-    ghc_args += ["-optP" + f for f in cc.cpp_flags]
++
++    # Write the -optP flags to a response because they can be very long on Windows
```

Comment:
Is this whole section only used on Windows? Not entirely obvious from the patch.

Reply:
No, we always use response files for -optP. Doesn't harm on Linux and makes it more consistent.